libs/core/src/components/pds-box/pds-box.tsx

```diff
@@ -13,6 +13,12 @@ export class PdsBox {
   */
   @Prop() alignItems?: `start` | `center` | `end` | `baseline` | `stretch`;
 
+  /**
+   * Defines the vertical alignment of the box items.
+   * @defaultValue start
+  */
+  @Prop() alignSelf?: `start` | `center` | `end` | `baseline` | `stretch`;
```

Comment:
I see the property defined, the styles in the scss, but I don't see any use of the property in this file. I would have expected to see this between lines [179](https://github.com/Kajabi/pine/pull/399/files#diff-1d93317147777b86f6188e858fb56eb8848ec4ef868869f3ae623b21c0b3c467R179) and [180](https://github.com/Kajabi/pine/pull/399/files#diff-1d93317147777b86f6188e858fb56eb8848ec4ef868869f3ae623b21c0b3c467R180)